frontend/src/app/header/header.component.html

```diff
@@ -1,27 +1,42 @@
 <div class="container">
   <nav class="navbar navbar-light navbar-expand-md">
-
-    <a class="navbar-brand" href="#">
+    <a class="navbar-brand" href="{{ djangoRoot }}">
       <img src="../../assets/img/volo_logo.svg" height="32" class="d-inline-block align-top" alt="">
     </a>
-
-    <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#mainNavbar"
-            aria-controls="mainNavbar" aria-expanded="false" aria-label="Toggle navigation">
+    <button class="navbar-toggler" type="button" (click)="isNavbarCollapsed = !isNavbarCollapsed" aria-controls="mainNavbar" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
-
-    <div class="navbar-collapse collapse justify-content-end" id="mainNavbar">
-      <ul class="navbar-nav mt-2 mt-md-0 align-items-end">
+    <div [ngbCollapse]="isNavbarCollapsed" class="collapse navbar-collapse justify-content-end" id="mainNavbar">
+      <ul *ngIf="!currentUser; else anonymousBlock" class="nav navbar-nav">
         <li class="nav-item">
-          <a href="{{djangoRoot}}/login" class="btn btn-primary my-2 my-sm-0">Załóż konto</a>
+          <a href="{{ djangoRoot }}/login" class="btn btn-primary my-2 my-sm-0">Załóż konto</a>
```

Comment:
{{ djangoRoot }} -> should that be with spaces within brackets? I've done it without them: 
https://github.com/CodeForPoznan/volontulo/pull/620/files

Reply:
I spoke with @elspawaczo and @Bosmanfrx and we decided to go with spaces there, because (as @Bosmanfrx said) "they look prettier"